metaflow/plugins/argo/argo_workflows.py

```diff
@@ -899,6 +910,10 @@ def _container_templates(self):
                 )
                 # Set emptyDir volume for state management
                 .empty_dir_volume("out")
+                # Set tmpfs emptyDir volume if enabled
+                .empty_dir_volume(
+                    "argo-tmpfs-volume", size_limit=tmpfs_size if tmpfs_enabled else 0
```

Comment:
should we call this - `tmpfs-ephemeral-volume` to make this consistent with @kubernetes